src/Totem.Tests/Features/Contracts/EditTests.cs

```diff
@@ -503,59 +447,6 @@ public async Task ShouldNotEditWhenContractStringTimestampDoesNotHaveFormat()
             command.ShouldNotValidate("The Timestamp property must have a format of date-time.");
         }
 
```

Comment:
SHOULD: Add integration test(s) for editing a contract removing Id, Timestamp, Type (null), and Namespace (null).

Reply:
Updated with f36a51801c338d9897cf50b74466bea3817029e8